### ISSUE TITLE
Working hours settings not saving

### ISSUE THREAD

**Issue by sborg-ms** [isaacrlevin/PresenceLight#72]
The new Light Syncing Schedule (working hours) does not seem to "save" the setting between session/re-starts of the app.

**Comment by sborg-ms** [comment]
It also seems once Configure Working Hours is checked - you cannot Uncheck it!

(not 100% related but regular/reliable syncing seems broken now as well)

**Comment by isaacrlevin** [comment]
Please try the latest nightly, as I have cleaned up the light sync process

**Comment by sborg-ms** [comment]
New nightly still seems to not be saving the working hours start/end time.  It also installed/started with the Configure Working hours pre-checked.

**Comment by isaacrlevin** [comment]
I know this is silly, but did you click save? I just tested Nightly and the the settings file is being updated. You can validate yourself, as the settings file is here

C:\Users\username\AppData\Local\Packages\37828IsaacLevin.197278F15330A.Local_dn7ec1ytrxd6a\LocalState\settings.json

**Comment by sborg-ms** [comment]
hi :) - Definitely checked and re-checked - it says Saved in green text! - confirming the settings.json file is in the nightly packages folder   (noticed the date format does not match my locale in case that is a issue here as we are DD/MM/YYYY ) - but it still does not show the values on restarting the app!

  "SyncLights": true,
  "UseWorkingHours": true,
  "WorkingHoursStartTime": "6/24/2020 8:00:38 AM",
  "WorkingHoursEndTime": "6/24/2020 6:00:53 PM",

**Comment by isaacrlevin** [comment]
can you show me a screenshot of your about page?

**Comment by sborg-ms** [comment]
Sure here you are attached.
![LightPresenceAbout](https://user-images.githubusercontent.com/67351620/85804045-f95a4000-b78b-11ea-9e40-11a1eab8d8aa.jpg)

**Comment by isaacrlevin** [comment]
I am running that version and my working hours are not being set as that, they look like below
```
  "SyncLights": true,
  "UseWorkingHours": true,
  "WorkingHoursStartTime": "15:00:00",
  "WorkingHoursEndTime": "18:15:00",
```

If you clear out the values in settings file like so

```
  "SyncLights": true,
  "UseWorkingHours": true,
  "WorkingHoursStartTime": "",
  "WorkingHoursEndTime": "",
```

That try again do you see the same issue

**Comment by sborg-ms** [comment]
cleared out as per above; updated to v2.0.75 and set my times 8am to 5:30pm and setting file still has the following

  "WorkingHoursStartTime": "6/27/2020 8:00:00 AM",
  "WorkingHoursEndTime": "6/27/2020 5:30:00 PM",

**Comment by sborg-ms** [comment]
Can we also add Days of Week  (testing this here on a Saturday got me thinking working hours definition should also have Days Of Week).

**Comment by isaacrlevin** [comment]
@sborg-ms can you record a video of an empty settings file being set the incorrect way?

**Comment by isaacrlevin** [comment]
Days of week added to latest nightly

**Comment by sborg-ms** [comment]
2.0.79.0 - cant test this as overall i cant get this version to work/sign-in/setup settings.

**Comment by isaacrlevin** [comment]
Fixed as of 2.0.85

**Comment by benlye** [comment]
In case anyone else stumbles across this - if you're still having this issue even with the latest builds, it's necessary to delete the old working hours settings from the settings.json file (with the app closed), then re-enabling working hours will create the start and end time values correctly.
